### PR TITLE
Include EOS CR theme

### PR DESCRIPTION
**Pull request by JustinCast**
## What does this PR do?
- EOS CR theme has been included in the template
## How should this be tested?
<code>yarn start</code>
go to http://localhost:3000

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
+			<Redirect to="/not-found" />
+		</Switch>
+	</BrowserRouter>
+);
 
 App.propTypes = {
-  ual: PropTypes.object
-}
+	ual: PropTypes.object
+};
```
Comment (adriexnet) [inline src/App.js:21]: We are using standardjs we should avoid semicolon https://standardjs.com/rules.html#semicolons

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
-      <Redirect exact from="/" to="/dashboard" />
-      <Redirect to="/not-found" />
-    </Switch>
-  </BrowserRouter>
-)
+	<BrowserRouter>
+		<Switch>
+			{routes.map(({ path, component: Component }) => (
+				<Route key={`path-${path}`} path={path}>
+					<Component ual={ual} />
```
Comment (adriexnet) [inline src/App.js:10]: with standardjs we should use 2 spaces for indentation https://standardjs.com/rules.html#rules

File: package.json [hunk 3]
```diff
@@ -1,65 +1,61 @@
...
-    "eslint-plugin-prettier": "^3.1.3",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-react": "^7.19.0",
-    "eslint-plugin-standard": "^4.0.1",
-    "husky": "^4.2.5",
-    "precise-commits": "^1.0.2",
-    "prettier": "^2.0.5",
-    "react-app-rewired": "^2.1.6"
-  }
+	"name": "eoscrwebappboilerplate",
```
Comment (adriexnet) [inline package.json:2]: To keep the same indentation across the project we are using prettier, please install https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode and intent this file with it

Comment (JustinCast) [reply package.json]: I think the wrong prettier version. Thanks!

**Pull request merged**